File: Programming_Task_02_Bhakti_Parhad/C/password_checker.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main()
{
    char password[100];

    int hasUpper = 0;
    int hasDigit = 0;
    int score = 0;

    printf("Enter Password: ");
    scanf("%s", password);

    for(int i = 0; password[i] != '\0'; i++)
    {
        if(isupper(password[i]))
            hasUpper = 1;

        if(isdigit(password[i]))
            hasDigit = 1;
    }

    if(strlen(password) >= 8)
        score++;

    if(hasUpper)
        score++;

    if(hasDigit)
        score++;

    if(score == 3)
        printf("Strong Password");
    else if(score == 2)
        printf("Moderate Password");
    else
        printf("Weak Password");

    return 0;
}

File: Programming_Task_02_Bhakti_Parhad/Python/password_checker.py
password = input("Enter Password: ")

has_upper = False
has_digit = False

for ch in password:
    if ch.isupper():
        has_upper = True

    if ch.isdigit():
        has_digit = True

score = 0

if len(password) >= 8:
    score += 1

if has_upper:
    score += 1

if has_digit:
    score += 1

if score == 3:
    print("Strong Password")
elif score == 2:
    print("Moderate Password")
else:
    print("Weak Password")